### PR TITLE
Dpcp

### PR DESCRIPTION
**Pull request by rafiw**

### CODE REVIEW

File: src/vma/dev/ib_ctx_handler.cpp [hunk 1]
```diff
@@ -54,84 +54,36 @@
...
+	, m_p_ibv_device_attr(NULL)
+	, m_p_ibv_pd(NULL)
+	, m_flow_tag_enabled(false)
 	, m_on_device_memory(0)
 	, m_removed(false)
 	, m_lock_umr("spin_lock_umr")
 	, m_umr_cq(NULL)
 	, m_umr_qp(NULL)
 	, m_p_ctx_time_converter(NULL)
+	, m_is_ok(false)
```
Comment (igor-ivanov) [inline src/vma/dev/ib_ctx_handler.cpp]: Having this flag does not guarantee correct check of class object creation. Exception can happen during initialization of one of the objects in initialization list. In this case `m_is_ok` can be in undefined state.

Comment (rafiw) [reply src/vma/dev/ib_ctx_handler.cpp]: all members get NULL and 0, if you want i can initialize the lick in the constractor body.

**Comment by igor-ivanov** [comment]
bot:retest

**Comment by igor-ivanov** [comment]
Duplicate https://github.com/Mellanox/libvma/pull/869

**Pull request closed without merging**